src/patternfly/components/ContextSelector/__context-selector-menu-menu.hbs

```diff
@@ -1,70 +1,70 @@
 {{#> context-selector-menu-menu}}
-  <li>
+  <li role="menuitem">
```

Comment:
For this file we'll want to update the `li` element to `role="none"`, then within the `context-selector-menu-list-item.hbs` component we want to add the `role="menuitem"`